libraries/bezier-rs/src/subpath/solvers.rs

```diff
@@ -138,7 +138,12 @@ impl<ManipulatorGroupId: crate::Identifier> Subpath<ManipulatorGroupId> {
 	}
 
 	/// Returns the manipulator point that is needed for a miter join if it is possible.
-	pub(crate) fn miter_line_join(&self, other: &Subpath<ManipulatorGroupId>) -> Option<ManipulatorGroup<ManipulatorGroupId>> {
+	/// - `miter_limit`: Defines a limit for the ratio between the miter length. When the limit is exceeded, a miter join will not be returned. This value should be greater than 1. If not, the default of 4 will be used.
```

Comment:
> ...ratio between the miter length ***[and what?]***.

Also, instead of saying "greater than 1", this should probably say "at least 1". Alternatively, change the `if miter_limit < 1. {` to `<=`.